libmamba/tests/src/core/test_mirror.cpp

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2023, QuantStack and Mamba Contributors
+//
+// Distributed under the terms of the BSD 3-Clause License.
+//
+// The full license is in the file LICENSE, distributed with this software.
+
+#include <doctest/doctest.h>
+
+#include "mamba/core/mirror.hpp"
+
+namespace mamba
+{
+    TEST_SUITE("mirror")
```

Comment:
I know it's planned but I note this as a future reminder: needs test, and one of the first one would be to implement a test mirror and make it work with the mechanism handling mirrors (no downloading required). Then for tests requiring some downloading, I suppose they will need to be somewhat optional?